pnpm-lock.yaml

```diff
@@ -38,7 +38,7 @@ importers:
       yootils: 0.3.1
     devDependencies:
       '@sveltejs/adapter-auto': 1.0.0-next.89
-      '@sveltejs/kit': 1.0.0-next.554_svelte@3.53.1+vite@3.2.4
+      '@sveltejs/kit': 1.0.0-next.555_svelte@3.53.1+vite@3.2.4
```

Comment:
was this on purpose or accidental because kit version is `next` in package.json?

Reply:
I guess pnpm just did that automatically. There shouldn't be any harm in it though